pep/app/authenticators/pep.ts

```diff
@@ -21,10 +21,10 @@ export default class PepAuthenticator extends BaseAuthenticator {
     async authenticate(username: string, password: string) {
         const params = serializeQueryParams({
             grant_type: 'password',
-            username,
-            password
+            UserName: username,
+            Password: password
         });
-        const result = await this.ajax.request(`Session/Login?${params}`);
+        const result = await this.ajax.request(`https://padstest.zedra.net/PEPSecure/api/v1/Authenticate?${params}`);
```

Comment:
you prob just didnt get to this yet, but we'll want to make this base url for the auth endpoints a new env var, like `AUTH_BASE_URL` and then import it here from `ENV`. to add new env vars you need to update a couple files:

- add it to every `.env-*` file in the root (leaving the value blank for staging/prod)
- pep/config/dotenv.js
- pep/app/config/environment.d.ts
- pep/config/environment.js